content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -566,60 +566,142 @@ it.skip('returns "fizz" when number is multiple of 3', () => {
 
 <Alert type="success">
 
-<Icon name="check-circle" color="green"></Icon> **Best Practice:** Clean up
-state **before** tests run.
+<Icon name="check-circle" color="green"></Icon> **Best Practice:** Tests should
+always be able to be run independently from one another **and still pass**.
 
 </Alert>
 
-Test isolation is the practice of resetting application state _before_ each
-test.
-
-Cleaning up state ensures that the operation of one test does not affect another
-test later on. The goal for each test should be to reliably pass whether run in
-isolation or consecutively with other tests. Having tests that depend on the
-state of an earlier test can potentially cause nondeterministic test failures.
-
-Cypress supports two modes of test isolation, `legacy` and `strict`.
+As stated in our mission, we hold ourselves accountable to champion a testing
+process that actually works, and have built Cypress to guide developer towards
+write independent tests from the start.
 
-#### Legacy Mode
+We do this by cleaning up state _before_ each test to ensure that the operation
+of one test does not affect another test later on. The goal for each test should
+be to **reliably pass** whether run in isolation or consecutively with other
+tests. Having tests that depend on the state of an earlier test can potentially
+cause nondeterministic test failures which makes debugging challenging.
 
-When in `legacy` mode, Cypress handles resetting the state for:
+Cypress will start each test with a clean test slate by restoring and clearing
+all:
 
 - [aliases](/api/commands/as)
-- [cookies](/api/commands/clearcookies)
-- [clock](/api/commands/clock)
-- [intercept](/api/commands/intercept)
-- [localStorage](/api/commands/clearlocalstorage)
+- [clock mocks](/api/commands/clock)
+- [intercepts](/api/commands/intercept)
 - [routes](/api/commands/route)
-- [sessions](/api/commands/session)
 - [spies](/api/commands/spy)
 - [stubs](/api/commands/stub)
-- [viewport](/api/commands/viewport)
+- [viewport changes](/api/commands/viewport)
+
+In additional to a clean test slate, Cypress also believes in run tests in a
+clean browser context such that the application or component under test behaves
+consistently when ran. This concept is described as `testIsolation`.
+
+The test isolation mode is a global configuration and can be overridden at the
+`describe` level with the
+[`testIsolation`](/guides/references/configuration#Global) option.
+
+#### End-to-end testing
+
+Cypress supports the following modes of test isolation in end-to-end testing to
+describe if a suite of tests should run in a clean browser context or not:
+`legacy` (deprecated), `on` (experimental) and `off` (experimental).
```

Comment:
It feels weird that the only options for test isolation are either `experimental` or `deprecated`. I get it, but it also feels like we don't officially support any of these. Wonder if there is a different way to brand this in the docs?

Reply:
I could use (current) but it'll be removed/changed in the next few months & wanted to call it out now. Thoughts on `current`?

Reply:
I am thinking `current` works until we release 11.0, in which we say it's removed. It sounds like we don't actually plan to deprecate it, just remove it.

Reply:
Is there any way we can just scrub legacy from a discussion point? I doubt very many people have even set it. Could we do something like, ignore the testIsolation setting (it's always legacy) unless experimental is enabled?